src/04_features/search/ui/search.scss

```diff
@@ -0,0 +1,56 @@
+@import '../../../01_app/styles/variables.scss';
```

Comment:
Do not need this import.  We have only one style file. All import should be in src/index.scss